beacon_node/network/src/sync/network_context/requests/blobs_by_root.rs

```diff
@@ -22,6 +22,7 @@ impl BlobsByRootSingleBlockRequest {
                 .collect(),
             spec,
         )
+        .expect("single block lookup should always satisfy limit for BlobsByRootRequest")
```

Comment:
I think the users of these methods return Results so i would definitely prefer propagating the errors

Reply:
oh yeah good call, you're right

fixing now

Reply:
Fixed in:

da692a6daf43fa5a61a550f2a39d5d98fa14ce06